src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
+    for="Products.CMFCore.interfaces.IFolderish"
+    factory=".upload.UploadOptions"
+    permission="cmf.AddPortalContent"
+    />
+
+  <plone:service
+    method="POST"
+    name="@upload"
+    for="Products.CMFCore.interfaces.IFolderish"
+    factory=".upload.UploadPost"
+    permission="cmf.AddPortalContent"
+    />
+
+  <plone:service
+    method="HEAD"
+    name="@upload"
+    for="Products.CMFCore.interfaces.IFolderish"
+    factory=".upload.UploadHead"
+    permission="cmf.AddPortalContent"
+    />
+
+  <plone:service
+    method="PATCH"
+    name="@upload"
+    for="Products.CMFCore.interfaces.IFolderish"
+    factory=".upload.UploadPatch"
+    permission="cmf.AddPortalContent"
+    />
 
+<!--   <plone:CORSPolicy
```

Comment:
Yes, but I would like to have it in tests and maybe in the docs. I'll work on it a little bit during the Easter weekend.